data_folder/config.yaml

```diff
@@ -28,10 +28,10 @@ date:
 positions:
   - Software engineer
 
-# One or more keywords to look for in the job description, for example, "chromium browser"
+# One or more keywords to look for in the job description, for example, "chromium browser". This field is optional.
 # Only use this field to find jobs that require specific technologies, products, or skills that is not mentioned in the job title.
 # Warning: Any jobs without matching keywords in the job description will be skipped.
-job_description_keywords:
+job_description_keywords_whitelist:
```

Comment:
can you make is as keywords whitelist

Reply:
I think it is worthy to be specific to make sure people understand the keywords apply to job description, not title.

Reply:
keywords_whitelist is in work_preference.yaml file -> job_description prefix is redundent provided the context of file.

Reply:
@49Simon , should I make this rename? I wouldn't mind either way.

Reply:
Done.